src/node_modules/app/components/elementary/ElasticSearchField.svelte

```diff
@@ -161,12 +159,19 @@
 		dispatch('change', event.detail);
 	}
 
-	// eslint-disable-next-line prefer-named-capture-group
-	const stringUnionPattern = /^(['"]\w*['"]\s*\|\s*)*['"]\w*['"]$/gu;
-	function getEditor (targetDataType) {
+	function isStringUnion (type) {
+		return Array.isArray(type) && type.every(subType => {
+			return subType === 'undefined'
+				|| typeof subType === 'string' && subType.startsWith('"');
+		});
+	}
```

Comment:
I'd use something like this (assuming `isStringType` makes sense):

```suggestion
	const isStringType = _.anyOf([
		_.is('undefined'),
		_.allOf([isString, makeStartsWith('"')])
	]);
	const isStringUnion = _.allOf([isArray, _.every(isStringType)]);
```